src/main/webapp/index.jsp

```diff
@@ -38,7 +38,12 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>CodeU Chat App</h1>
-      <h2>Welcome!</h2>
+        <h2>Welcome to ~Team 31~'s chat app!</h2>
```

Comment:
This looks over indented from `<h1>` now. The `<p>` contents looks good now, though! The whole block should still be at the same document level as the the `<h1>`

That is:
```
<h1>CodeU Chat App</h1>
<h2>Welcome to ~Team 31~'s chat app!</h2>
<p>
  Lorem ipsum dolor...
</p>
```